rpdc.py

```diff
@@ -533,3 +539,8 @@ def validateJSONConfigContent(variantConfig):
             deleteBaseAsset(uploadURLs['id'], accessToken)
             for rapidModelID in newRapidModelIDs:
                 deleteRapidModel(rapidModelID, accessToken)
+
+# Exit with error
+if(exitOnError and failedOptimizations > 0):
+    print("Exiting with error because of failed optimizations")
+    sys.exit(42)
```

Comment:
Is this a specific error code?

Reply:
No, randomly picked. sys.exit() without a code didnt trigger an error on the pipeline, so I picked 42

Reply:
I would suggest defining specific error codes for different errors conditions defined as "constants":
```
OPTIMIZATION_FAILED_ERROR = 2
SOME_OTHER_ERROR = 3
(...)
```

Reply:
Thanks, added that